src/output.jl

```diff
@@ -39,9 +44,12 @@ plots the habitat suitability of a landscape for a given timestep t
 """
 function image_suitability(SD::Simulation_Data, t::Int)
```

Comment:
![grafik](https://github.com/janablechschmidt/MetaRange.jl/assets/77731816/c45d9a19-32af-40a0-b06c-71839a2bd469)
Returns image in wrong orientation

Reply:
done!